src/mudlet.pro

```diff
@@ -187,6 +187,13 @@ isEmpty( 3DMAPPER_TEST ) | !equals(3DMAPPER_TEST, "NO" ) {
     DEFINES += INCLUDE_3DMAPPER
 }
 
+
+# Some temporary tests to see whether either of these are present in the
+# AppVeyor CI environment, the second is true on a local Desktop MSYS2 Windows
+# OS:
+cygwin : message("A 'cygwin' qmake scope is defined.")
+mingw : message("A 'mingw' qmake scope is defined.")
```

Comment:
This can go now, right?

Reply:
Yeah, for the record: `mingw` (but not `mingw32`/`mingw64` - which is a pain for other reasons) is present on both my local MSYS2 setup and during the AppVeyor one.

It may also be useful to note that `linux` ***is*** a valid scope for `qmake` now as well - not sure how far back that goes. *Conversely there is not a CMake equivalent to `WIN32` for Linux - you have to consider the value of CMAKE_SYSTEM_NAME instead but I digress and THAT was really more useful to find out for #3515 ...!*

Reply:
Removed.